src/sentry/ratelimits/base.py

```diff
@@ -26,8 +26,8 @@ def current_value(
 
     def is_limited_with_value(
         self, key: str, limit: int, project: Project | None = None, window: int | None = None
-    ) -> tuple[bool, int]:
-        return False, 0
+    ) -> tuple[bool, int, int]:
+        return False, 0, 0
```

Comment:
what is this change for?

Reply:
make sure the parent class and the child class have the same return types